client/driver/docker.go

```diff
@@ -1924,6 +1933,25 @@ func (h *DockerHandle) run() {
 		h.logger.Printf("[ERR] driver.docker: failed to inspect container %s: %v", h.containerID, ierr)
 	} else if container.State.OOMKilled {
 		werr = fmt.Errorf("OOM Killed")
+		labels := []metrics.Label{
+			{
+				Name:  "JobName",
+				Value: h.jobName,
+			},
+			{
+				Name:  "TaskGroupName",
+				Value: h.taskGroupName,
+			},
+			{
+				Name:  "TaskName",
```

Comment:
@jippi this is what you meant right?

Reply:
yeah

I personally don't care for the allocation id below, its a very high cardinality field, making influxdb & friends unhappy fairly fast

Reply:
I understand that from the database and query point of view it probably has no value but from the alerting point of view the user can quickly run `nomad alloc-status <alloc-id>` and see what is happening.

What would you say here? Should we keep it or drop it.

Reply:
my personal preference would be to drop it - sounds like an alert in your logging rather than telemetry if you ask me :)

Reply:
I would drop it. As @jippi points out it will make many telemetry systems unhappy